### PR TITLE
PokeDex and PokeDex Detail pages changed/fixed

### PR DESCRIPTION
**Pull request by philbu**
- Minor fixes in the css of PokeDex Detail
- Fixed loading issue in PokeDex

### CODE REVIEW

File: ionic2/app/pages/poke-detail/poke-detail.page.scss [hunk 1]
```diff
@@ -3,12 +3,12 @@
 }
```
Comment (WoH) [inline ionic2/app/pages/poke-detail/poke-detail.page.scss:7]: We should add a light grey background color (as in the Poke(in)Dex)

File: ionic2/app/pages/wiki-index/wiki-index.page.ts [hunk 2]
```diff
@@ -12,17 +12,22 @@ import { ApiService } from '../../services/api.service';
...
 
   queryString: string;
   querySubscription: Subscription;
   results: Pokemon[] = [];
 
   constructor(private navCtrl: NavController, private apiservice: ApiService) { }
 
-  ngOnInit() {
-    this.querySubscription = this.apiservice.getAllPokemon()
+  ionViewDidEnter() {
```
Comment (WoH) [inline ionic2/app/pages/wiki-index/wiki-index.page.ts:23]: I found viewLoaded to be better. Also, I fixed that in my fix all the things PR

**Comment by WoH** [comment]
If you solve the merge conflict and keep my new hook intact this should be good to go.

Edit: Please change the background color though.

**Comment by Lugitan** [comment]
badges_test_v2

I did a bit of styling and fixed the badges in the Pokémon details view. I created two options for the badges. First one is done with the `<ion-badge>` tag as before. For the second one the tag has been removed and will be just a `<div>` then.
Any Preferences?

**Comment by WoH** [comment]
2/3 with the badge color corresponding to the the label color

**Comment by johartl** [comment]
I like the div better. However, can you make the width of the badge a few pixels smaller.

**Comment by Lugitan** [comment]
Sure. The dimensions are a bit weird anyway. I will look into it this evening.

File: ionic2/app/pages/poke-detail/poke-detail.page.scss (hunk 1 above)
Comment (philbu) [reply ionic2/app/pages/poke-detail/poke-detail.page.scss:7]: Done

File: ionic2/app/pages/wiki-index/wiki-index.page.ts (hunk 2 above)
Comment (philbu) [reply ionic2/app/pages/wiki-index/wiki-index.page.ts:23]: Is your change already live? Didn't display a merge conflict with ionViewDidEnter

**Comment by philbu** [comment]
Fixed the new image sources. It isn't .icon anymore. Instead it is .gifIcon, .pngIcon and .svgIcon. Unfortunately the data team hasn't uploaded pictures for svgIcon yet and the pngIcons aren't highres right now.

File: ionic2/app/models/pokemon.ts [hunk 3]
```diff
@@ -59,8 +59,16 @@ export class Pokemon {
...
-    return config.apiEndpoint + '/api/pokemon/id/' + this.pokemonId + '/icon';
+  get gifIcon(): string {
+    return config.apiEndpoint + '/api/pokemon/id/' + this.pokemonId + '/icon/gif';
+  }
+
+  get pngIcon(): string {
+    return config.apiEndpoint + '/api/pokemon/id/' + this.pokemonId + '/icon/png';
+  }
+
+  get scgIcon(): string {
```
Comment (AlexanderLill) [inline ionic2/app/models/pokemon.ts]: @philbu here is a typo :)
- [x] rename "scg" -> "svg"

File: ionic2/app/pages/wiki-index/wiki-index.page.ts [hunk 4]
```diff
@@ -21,8 +21,13 @@ export class WikiIndexPage {
   constructor(private navCtrl: NavController, private apiservice: ApiService) { }
 
   ionViewDidEnter() {
```
Comment (AlexanderLill) [inline ionic2/app/pages/wiki-index/wiki-index.page.ts]: Do we have a use case where this query string might be pre-filled? Are we doing this when we jump back from the detail view?

**Review by AlexanderLill — CHANGES_REQUESTED** [review]

File: ionic2/app/models/pokemon.ts (hunk 3 above)
Comment (philbu) [reply ionic2/app/models/pokemon.ts]: Thanks. Fixed it. :+1:

**Review by AlexanderLill — APPROVED** [review]

**Comment by johartl** [comment]
Is this good to merge?

**Comment by johartl** [comment]
@philbu I will merge this now. I really need some of the changes in here to finish the POI card.

**Pull request merged**